cdc/sinkv2/eventsink/txn/worker.go

```diff
@@ -86,8 +96,11 @@ func (w *worker) runBackgroundLoop() {
 }
 
 func (w *worker) doFlush() bool {
-	if err := w.backend.Flush(); err != nil {
-		// TODO: handle err.
+	// TODO: support to cnacel the worker in flushing.
+	ctx := context.Background()
+	if err := w.backend.Flush(ctx); err != nil {
+		log.Warn("txn sink worker flush fail", zap.Error(err))
+		w.errCh <- err
```

Comment:
Can this line block indefinitely?

Reply:
Yes it does. I think it's expected because if the error is unrecoverable, we can't drop it silently.